experiments/simbricks/orchestration/nodeconfig.py

```diff
@@ -26,7 +26,7 @@
 import tarfile
 import typing as tp
 
-from simbricks.orchestration.experiment.experiment_environment import ExpEnv
+import simbricks.orchestration.experiment.experiment_environment as env
```

Comment:
```suggestion
from simbricks.orchestration.experiment import experiment_environment as exp_env
```
This was unfortunately still the wrong form. Have a look at this section here: https://google.github.io/styleguide/pyguide.html#224-decision.

Also, I would have kept the variable name as `env` and then rename the imported module to `exp_env` but choose whatever you find better here.